### PR TITLE
Fix checkstyle for EOstdin$EOφ

### LINKED ISSUE #882: Fix checkstyle warnings for `EOstdin$EOφ`
Split from https://github.com/objectionary/eo/issues/739

Refactor the class to pass all checkstyle checks.
Current warnings:
```
[INFO] Checkstyle: src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java[58]: Return count is 2 (max allowed for non-void methods/lambdas is 1). (ReturnCountCheck)
[INFO] Checkstyle: src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java[62]: Name 'b' must match pattern '^(id|[a-z]{3,12})$'. (LocalFinalVariableNameCheck)
[INFO] Checkstyle: src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java[69]: Parameter e should be final. (FinalParametersCheck)
[INFO] Checkstyle: src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java[69]: Name 'e' must match pattern '^(ex|[a-z]{3,12})$'. (CatchParameterNameCheck)
```

### PR DESCRIPTION
**Pull request by alex-semenyuk**
Close #882

### CODE REVIEW

**Comment by Graur** [comment]
@alex-semenyuk If I understand correctly, the `ReturnCountCheck` didn't fixed here. Or I am wrong?

**Comment by alex-semenyuk** [comment]
Seems It was fixed before. Now I see only this one issue in the file.

**Review by Graur — APPROVED** [review]

**Comment by Graur** [comment]
@rultor merge

**Comment by Graur** [comment]
@rultor stop

File: eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java [hunk 1]
```diff
@@ -58,11 +58,11 @@
                     try (BufferedInputStream bis = new BufferedInputStream(System.in);
                         ByteArrayOutputStream buf = new ByteArrayOutputStream()) {
                         while (true) {
-                            final int b = bis.read();
-                            if (b == -1) {
+                            final int byteData = bis.read();
```
Comment (Graur) [inline eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java:61]: @alex-semenyuk Can you make this name not compound? Please, read this: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

**Review by Graur — CHANGES_REQUESTED** [review]

File: eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java (hunk 1 above)
Comment (alex-semenyuk) [reply eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java:61]: Yeah, I read this, but I think here more suitable is byte, but it's java keyword.

Comment (Graur) [reply eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java:61]: @alex-semenyuk I think `data` is good enough

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@alex-semenyuk thanks!

**Pull request merged**